docs/source/common/test_set.rst

```diff
@@ -1,17 +1,26 @@
 .. _test_set:
 
-Test set
-========
```

Comment:
This page is a bit confusing. We should rename it `evaluation.rst`.

Reply:
yes. forgot to mention. will be doing that once approved for better review now.

Reply:
won't that break existing links?

Reply:
yes.. not updating now. will be renaming them at the very end once all docs are done.